rdr/busybox/deploymnet/mysql.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: apps/v1 # for versions before 1.9.0 use apps/v1beta2
+kind: Deployment
+metadata:
+  name: mysql
+spec:
+  selector:
+    matchLabels:
+     workload : mysql
+  strategy:
+    type: Recreate
+  template:
+    metadata:
+      labels:
+        workload : mysql
+    spec:
+      containers:
+      - image: quay.io/prsurve/mysql:latest
+        name: mysql
+        env:
+          # Use secret in real usage
+        - name: MYSQL_ROOT_PASSWORD
+          value: password
```

Comment:
same as above, replace the password with placeholder

Reply:
user does not have to be set any values here some values are static and some are dynamically genered

Reply:
passwords can not be stored as plain text on github